runners/spark/src/main/java/org/apache/beam/runners/spark/TestSparkPipelineOptions.java

```diff
@@ -32,4 +36,22 @@
   boolean isForceStreaming();
   void setForceStreaming(boolean forceStreaming);
 
+  @Description("A hard-coded expected number of assertions for this test pipeline.")
+  @Nullable
+  Integer getExpectedAssertions();
+  void setExpectedAssertions(Integer expectedAssertions);
+
+  @Description("A customizable EOT watermark in Millis.")
```

Comment:
Again, relating to my previous comment, I'd like to be able to terminate the pipeline on "end-of-time" watermark. But for the first execution I'd like to do it "mid-life" to simulate a failure/stop. so the watermark is not at infinity yet.

I really don't know why I decided to save words in documentation :upside_down_face:, I'll fix that.